mmdet/datasets/pipelines/loading.py

```diff
@@ -37,9 +37,11 @@ class LoadImageFromFile:
     def __init__(self,
                  to_float32=False,
                  color_type='color',
+                 channel_order='bgr',
```

Comment:
add description for `channel_order` in docstring

Reply:
please add unit-test for `channel_order`